src/pspm_cfg/pspm_cfg_split_sessions.m

```diff
@@ -8,16 +8,34 @@
 
 %% Specific items
 
+%% prefix
+split_prefix        = cfg_entry;
+split_prefix.name   = 'Prefix (data before split point)';
+split_prefix.tag    = 'prefix';
+split_prefix.strtype = 'i';
+split_prefix.num    = [1 1];
+split_prefix.val    = {0};
+split_prefix.help   = pspm_cfg_help_format('pspm_split_sessions', 'options.prefix');
+
+%% suffix
+split_suffix        = cfg_entry;
```

Comment:
Tested this. I feel this suffix does seem to check if it is larger than the length of data?
<img width="817" alt="image" src="https://github.com/user-attachments/assets/7d00c1c2-2bde-4cc5-b455-2b04b49a0fde">

Reply:
`pspm_split_sessions` uses `pspm_trim` which does not trim if `suffix` is longer than the remaining data.